### PR TITLE
feature 1 took me half an hour

### PR DESCRIPTION
**Pull request by Arsalan74**
It took me half an hour 
and I made a pull request to feature 2 branch

### CODE REVIEW

File: todo-list/src/components/App/index.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    task:"item3",
+    id:1
+  }
+];
+
+const App = () => {
+  return (
+  list.map((item)=> {
+    return (
+      <div>
```
Comment (sassanh) [inline todo-list/src/components/App/index.js:23]: @Arsalan74 Each react element in an array needs a key, please research about it and explain here why `key` prop is needed in this case.

**Comment by Arsalan74** [comment]
I think React is all about performance and it always re-renders unchanged elements. If you assign each element an unique and static key, React can keep track of changed elements and only re-render them.

P.S: Sorry for the late response, I didn’t notice your email last night

**Comment by sassanh** [comment]
Correct! Please update your PR accordingly so that react elements in the array have keys.

**Comment by Arsalan74** [comment]
I did so

**Comment by Arsalan74** [comment]
I did so,

**Pull request closed without merging**